test/integration/redirects_test.rb

```diff
@@ -77,4 +78,67 @@ def test_destroy_article_put
     assert_equal(article_url(Article.first.id),
                  @response.header["Location"])
   end
+
+  #######
```

Comment:
**May I make multiple requests per method?**
I wrote this "request test" style -- one request per test method. It could be be much DRYer -- and someone more efficient -- if I could do multiple requests per method.

Reply:
FWIW, I'm (obviously!) fine with multiple requests per test.

Reply:
The code here looks reasonable to me.  I'm not a real hardliner on only have one request per test or what some argue, that there should only be one assert per test.  However, having separate specs for each redirect seems right to me.  The way I think about it is if one of them fails does that mean all of them should fail?  If the requests are part of a coherent whole then it makes sense to me to have them together.  In this case I don't think these meet that bar.  I could easily see anyone of these failing with it having no implications on the other ones failing.  If that happens and these are all together, then it can be a problem since the first one to fail wins and you don't get any information about the other ones that are effectively independent.

Reply:
Thanks. I'm good with that. (And right now, it's less work.)